microsoftgraph/microsoft-graph-explorer-v2
# Feature: minimise sidebar

## Overview

Collapses and expands the sidebar menu to increase the size of the screen.

Closes #310 

### Demo

Standard (Anonymous):
![image](https://user-images.githubusercontent.com/58787602/72071215-57898880-32fc-11ea-8e6e-a4df45e7dc08.png)

Minimised (Anonymous):
![image](https://user-images.githubusercontent.com/58787602/72071249-68d29500-32fc-11ea-92df-f65677272904.png)

Standard (Logged In)
![image](https://user-images.githubusercontent.com/58787602/72071358-a9321300-32fc-11ea-97ec-c32644912cf8.png)

Minimised (Logged in)

![image](https://user-images.githubusercontent.com/58787602/72071635-4c832800-32fd-11ea-8f08-9541adcff0e8.png)

### Notes

Optional. Ancillary topics, caveats, alternative strategies that didn't work out, anything else.

## Testing Instructions

* Click the hamburger icon
* Notice the size of the sidebar changes
